contentcuration/contentcuration/frontend/shared/data/serverSync.js

```diff
@@ -353,23 +490,73 @@ let intervalTimer;
 export function startSyncing() {
   // Initiate a sync immediately in case any data
   // is left over in the database.
-
+  debouncedSyncChanges();
```

Comment:
I think it would be nice if we decrease `SYNC_IF_NO_CHANGES_FOR` which is the debounce time to something less than the existing 2 seconds, perhaps 500ms?